libs/hwes/eventstoredb/entity_event.go

```diff
@@ -0,0 +1,63 @@
+package eventstoredb
+
+import (
+	"context"
+	"fmt"
+	"telemetry"
+
+	"github.com/EventStore/EventStore-Client-Go/v4/esdb"
+	"github.com/google/uuid"
+	"google.golang.org/protobuf/proto"
+	"hwes"
+)
+
+type EntityAggregate struct {
+	*hwes.AggregateBase
+}
+
+func NewEntityAggregate(entityType string, entityId uuid.UUID) *EntityAggregate {
+	a := &EntityAggregate{}
+	a.AggregateBase = hwes.NewAggregateBase(hwes.AggregateType(entityType), entityId)
+	return a
+}
+
+func (a *EntityAggregate) GetTypeID() string {
+	return fmt.Sprintf("entity_%s-%s", a.GetType(), a.GetID())
+}
+
+// SaveEntityEvent creates a temporary EntityAggregate to create
```

Comment:
not relevant for the devs using the function (i think? or does this cause some side effect?)